neuralmonkey/decoders/decoder.py

```diff
@@ -19,7 +21,7 @@
 # pylint: disable=too-many-instance-attributes,too-few-public-methods
 # Big decoder cannot be simpler. Not sure if refactoring
 # it into smaller units would be helpful
-class Decoder(object):
+class Decoder(ModelPart):
```

Comment:
Ale ani nevim, co navrhnout.. ``Coder`` by bylo hezký, ale asi by to mohlo někoho mást